### PR TITLE
make it so it doesn't say I don't know what's going on when you quit,…

### PR DESCRIPTION
**Pull request by kessbethler**
… implement quit command

### CODE REVIEW

**Comment by Pyromuffin** [comment]
[re-implement quit and die](https://trello.com/c/ZfPx4V4o/12-re-implement-quit-and-die)

**Comment by kessbethler** [comment]
[make it so that we don't get "i don't know what is gong on" when we quit](https://trello.com/c/5qxzmKVQ/13-make-it-so-that-we-don-t-get-i-don-t-know-what-is-gong-on-when-we-quit)

File: commands.c [hunk 1]
```diff
@@ -82,6 +83,12 @@ void ExecuteMoveCommand(const Command* this, Referent* subject, Referent* object
     }
 }
 
+void ExecuteQuitCommand(const Command* this, Referent* subject, Referent* object)
+{
+    printf("Goodbye.\n");
```
Comment (Pyromuffin) [inline commands.c]: I like this

Comment (Pyromuffin) [inline commands.c:88]: I like this

**Review by Pyromuffin — APPROVED** [review]

File: state.h [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+// Created by Beth Kessler on 5/4/17.
+//
+
+#pragma once
+
+#include <stdbool.h>
+
+typedef enum ProgramRunningMode
+{
+    kInProgress,
```
Comment (Pyromuffin) [inline state.h]: I would prefer a more descriptive name than kInProgress.

Comment (Pyromuffin) [reply state.h]: I'm not sure what goes here!!

**Review by Pyromuffin — CHANGES_REQUESTED** [review]

**Review by Pyromuffin — APPROVED** [review]

**Review by Pyromuffin — APPROVED** [review]

**Pull request merged**